[2025-01-05T22:12:56.459+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-05T22:12:56.497+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kaggle_dag.end_task manual__2025-01-05T22:12:39.202851+00:00 [queued]>
[2025-01-05T22:12:56.515+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kaggle_dag.end_task manual__2025-01-05T22:12:39.202851+00:00 [queued]>
[2025-01-05T22:12:56.516+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-05T22:12:56.549+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): end_task> on 2025-01-05 22:12:39.202851+00:00
[2025-01-05T22:12:56.561+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=149) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-05T22:12:56.565+0000] {standard_task_runner.py:72} INFO - Started process 150 to run task
[2025-01-05T22:12:56.566+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'kaggle_dag', 'end_task', 'manual__2025-01-05T22:12:39.202851+00:00', '--job-id', '766', '--raw', '--subdir', 'DAGS_FOLDER/kaggle_dag.py', '--cfg-path', '/tmp/tmp6ctkm7lg']
[2025-01-05T22:12:56.569+0000] {standard_task_runner.py:105} INFO - Job 766: Subtask end_task
[2025-01-05T22:12:56.663+0000] {task_command.py:467} INFO - Running <TaskInstance: kaggle_dag.end_task manual__2025-01-05T22:12:39.202851+00:00 [running]> on host ffbd28c5220f
[2025-01-05T22:12:56.864+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='kaggle_dag' AIRFLOW_CTX_TASK_ID='end_task' AIRFLOW_CTX_EXECUTION_DATE='2025-01-05T22:12:39.202851+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-05T22:12:39.202851+00:00'
[2025-01-05T22:12:56.868+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-05T22:12:56.915+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-01-05T22:12:56.918+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Here we end! "']
[2025-01-05T22:12:56.938+0000] {subprocess.py:99} INFO - Output:
[2025-01-05T22:12:56.941+0000] {subprocess.py:106} INFO - Here we end!
[2025-01-05T22:12:56.942+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-01-05T22:12:57.012+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-05T22:12:57.014+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=kaggle_dag, task_id=end_task, run_id=manual__2025-01-05T22:12:39.202851+00:00, execution_date=20250105T221239, start_date=20250105T221256, end_date=20250105T221257
[2025-01-05T22:12:57.106+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-05T22:12:57.144+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-05T22:12:57.150+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
